client/components/inventory.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+var Inventory = (props) => {
+
+  return (
+    <div class="size" title="{props.count}">
```

Comment:
If you destructure as shown above rather than using title={props.count} you can use title={count}